src/main.cpp

```diff
@@ -1145,6 +1145,8 @@ int main(int argc, char **argv) {
 
                 if (cmd == CmdRun) {
                     ZigList<const char*> args = {0};
+                    args.append(buf_ptr(buf_out_name));
```

Comment:
shouldn't this be `exec_path` (below)?

Reply:
exec_path contains the full path of the application.

`man exec` says:

> The execv(), execvp(), and execvP() functions provide an array of pointers to null-terminated strings that represent the argument list available to the new program.  The first argument, by convention, should point to the file name associated with the file being executed.

Reply:
Seems to match `exec_path` then right? That's the file name of the file being executed.

Reply:
Whoops. I was under the impression that argv[0] was just the binary name. I didn't know that was not the case. Fixing with a new commit.